ldk/goja/.eslintrc.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
     'class-methods-use-this': 'off',
     'import/no-extraneous-dependencies': ['error', { devDependencies: true }],
     'import/prefer-default-export': 'off', // Default exports are bad.
+    'no-console': 'off',
```

Comment:
It has occurred to me that these console logs will appear in our unit test output as the rejection tests execute. We will need to come up with a more reliable way to do these logs.

Reply:
I wonder if we can mock the console functions... if so we can do that before each test (in a shared before each or something)

Reply:
We could abstract console perhaps...

Reply:
Could potentially mock console.log() globally like with olivehelps. I could give this a shot with the window aptitude card - if it doesn't pan out I will make a task for us to figure out a nicer way.